components/hab/src/exec.rs

```diff
@@ -77,8 +77,8 @@ where
         return Err(Error::ExecCommandNotFound(command));
     }
 
-    let fs_root_path = Path::new("/");
-    match PackageInstall::load_at_least(ident, None) {
+    let fs_root_path = Path::new(&*FS_ROOT_PATH);
```

Comment:
How about:
```rust
    let fs_root_path = FS_ROOT_PATH.as_path();
```

I think doing this more explicitly is easier to understand for those who may not intuitively understand all of the rust magic going on here. Especially when the name `FS_ROOT_PATH` doesn't make it clear that its type is actually a reference to a `PathBuf`.

```rust
    let fs_root_path = &*FS_ROOT_PATH;
```
Works too, but I think that while it's shorter, it's more confusing.